packages/smarthr-ui/src/components/ComboBox/SingleComboBox/stories/VRTSingleComboBox.stories.tsx

```diff
@@ -0,0 +1,153 @@
+    label: 'option 1',
+    value: 'value-1',
+    data: {
+      name: 'test',
+      age: 23,
+    },
+  },
+  'option 2': {
+    label: 'option 2',
+    value: 'value-2',
+    data: {
+      name: 'test 2',
+      age: 34,
+    },
+  },
+  'option 3': {
+    label: 'option 3',
+    value: 'value-3',
+    disabled: true,
+  },
+  'option 4': {
+    label: 'option 4',
+    value: 'value-4',
+  },
+  'option 5': {
+    label: 'option 5',
+    value: 'value-5',
+  },
+  'アイテムのラベルが長い場合（ダミーテキストダミーテキストダミーテキストダミーテキスト）': {
+    label: 'アイテムのラベルが長い場合（ダミーテキストダミーテキストダミーテキストダミーテキスト）',
+    value: 'value-6',
+  },
+  アイテムのラベルがReactNodeの場合: {
+    label: (
+      <Stack as="span" gap={0.25}>
+        <span>アイテムのラベルがReactNodeの場合</span>
+        <span>（ダミーテキストダミーテキストダミーテキストダミーテキスト）</span>
+      </Stack>
+    ),
+    value: 'value-7',
+  },
+}
+
+const prefixes = { なし: '', あり: <FaCirclePlusIcon /> }
+
+/* pict singleComboBox.pict
+ * disabled        error   width   prefix  selectedItem
+ * false   false   なし    なし    あり
+ * false   true    あり    あり    なし
+ * true    true    あり    なし    あり
+ * true    false   あり    あり    なし
+ * true    true    なし    なし    なし
+ * true    false   なし    あり    あり
+ */
+
+const _cases: Array<Omit<Parameters<typeof SingleComboBox>[0], 'items'>> = [
+  {
+    disabled: false,
+    error: false,
+    width: undefined,
+    prefix: undefined,
+    selectedItem: defaultItems['option 1'],
+  },
+  { disabled: false, error: true, width: '15em', prefix: prefixes['あり'], selectedItem: null },
+  {
+    disabled: true,
+    error: true,
+    width: '15em',
+    prefix: undefined,
+    selectedItem:
+      defaultItems[
+        'アイテムのラベルが長い場合（ダミーテキストダミーテキストダミーテキストダミーテキスト）'
+      ],
+  },
+  { disabled: true, error: false, width: '15em', prefix: prefixes['あり'], selectedItem: null },
+  { disabled: true, error: true, width: undefined, prefix: undefined, selectedItem: null },
+  {
+    disabled: true,
+    error: false,
+    width: undefined,
+    prefix: prefixes['あり'],
+    selectedItem: defaultItems['アイテムのラベルがReactNodeの場合'],
+  },
+]
+
+export default {
+  title: 'Forms（フォーム）/SingleComboBox/VRT',
+  component: SingleComboBox,
+  render: (args) => (
+    <Stack align="flex-start" gap={2}>
+      <form>
+        <FormControl className="shr-mb-[15rem]" title="デフォルト" dangerouslyTitleHidden>
+          <SingleComboBox
+            name="default"
+            items={Object.values(defaultItems)}
+            dropdownHelpMessage="入力でフィルタリングできます。"
+            selectedItem={null}
+          />
+        </FormControl>
+      </form>
+      <Stack align="flex-start">
+        {_cases.map((props, i) => (
+          <form key={i}>
+            <FormControl title="VRT 用の Story です" dangerouslyTitleHidden>
+              <SingleComboBox {...args} {...props} items={Object.values(defaultItems)} />
+            </FormControl>
+          </form>
+        ))}
+      </Stack>
+    </Stack>
+  ),
+  args: {
+    items: Object.values(defaultItems),
+  },
```

Comment:
default に書いてしまってよさそう。

```suggestion
  },
  play: playSingle,
```